feedTheMonster.ts

```diff
@@ -54,12 +61,15 @@ class App {
   }
 
   private async init() {
+    console.log("init");
     const font = await Utils.getLanguageSpecificFont(this.lang);
+    console.log(font, this.lang);
     await this.loadAndCacheFont(font, `./assets/fonts/${font}.ttf`);
     await this.preloadGameAudios();
     this.handleLoadingScreen();
     this.setupCanvas();
     const data = await getData();
+    console.log(data);
```

Comment:
Can we remove these console logs?

Reply:
@bernhardccodev remove this console. please check